core/network/common.hpp

```diff
@@ -21,6 +21,7 @@ namespace kagome::network {
   const libp2p::peer::Protocol kBlockAnnouncesProtocol =
       "/{}/block-announces/1";
   const libp2p::peer::Protocol kGrandpaProtocol = "/paritytech/grandpa/1";
+  const libp2p::peer::Protocol kGrandpaProtocolTemplate = "/{}/grandpa/1";
```

Comment:
Let's rename constant of old protocol name to **kGrandpaProtocolOld**, and new add as **kGrandpaProtocol**
```suggestion
  const libp2p::peer::Protocol kGrandpaProtocolOld = "/paritytech/grandpa/1";
  const libp2p::peer::Protocol kGrandpaProtocol = "/{}/grandpa/1";
```